kolibri/core/auth/test/test_bulk_import.py

```diff
@@ -150,18 +153,18 @@ def import_exported_csv(self):
             assert len(classroom.get_coaches()) == 1
 
     def test_dryrun_from_export_csv(self):
-        with open_csv_for_reading(self.filepath) as source:
-            header = next(csv.reader(source, strict=True))
+        source = file_storage.open(self.filepath, "r")
```

Comment:
Looking back at this, the reassignment of `source` makes this a it odd to read in a way that it wasn't when they were created inside of `with` statements.

I'll come back through and do some cleaning